app/src/main/kotlin/com/hippo/ehviewer/ui/tools/Dialog.kt

```diff
@@ -281,14 +281,32 @@ class DialogState {
         onCancelButtonClick: () -> Unit = {},
         secure: Boolean = false,
         text: @Composable (() -> Unit)? = null,
+    ) = awaitConfirmationOrCancel(
+        confirmButton = {
+            TextButton(onClick = it, enabled = confirmButtonEnabled) {
```

Comment:
The `confirmButtonEnabled` parameter can be removed as the new overload is added.
Though if we want to move clear igneous to the dialog in the future, it's better to use a no button dialog and place the reset button inside.

Reply:
Done